docs/content/commands/generate-include-scripts.md

```diff
@@ -0,0 +1,55 @@
+## Generating include scripts for all Nuget packages
+
+It is possible to generate include scripts for all registered Nuget packages defined in paket.dependencies.
+
+    [lang=batchfile]
+    $ paket generate-include-scripts framework net45
+
+This will create .csx and .fsx scripts under `paket-files/include-scripts/net45/`, those files can now be 
+used in your scripts without having to bother with the list and order of all dependencies for given package.
+
+Note: this command only works after packages have been restored, please call `paket restore` before using `paket generate-include-scripts` or `paket install` if you just changed your `paket.dependencies` file.
```

Comment:
I think create script should do a restore.
I think gc should be extended to clean unused load scripts
On May 10, 2016 10:37, "Gauthier Segay" <notifications@github.com> wrote:

> In docs/content/commands/generate-include-scripts.md
> <https://github.com/fsprojects/Paket/pull/1619#discussion_r62631374>:
>
> > @@ -0,0 +1,55 @@
> > +## Generating include scripts for all Nuget packages
> > +
> > +It is possible to generate include scripts for all registered Nuget packages defined in paket.dependencies.
> > +
> > +    [lang=batchfile]
> > +    $ paket generate-include-scripts framework net45
> > +
> > +This will create .csx and .fsx scripts under `paket-files/include-scripts/net45/`, those files can now be
> > +used in your scripts without having to bother with the list and order of all dependencies for given package.
> > +
> > +Note: this command only works after packages have been restored, please call `paket restore` before using `paket generate-include-scripts` or `paket install` if you just changed your `paket.dependencies` file.
>
> You mean having the command itself call restore to remove the need from
> the user?
>
> If you think that's the sensible choice, yes I can make a PR for this.
>
> Similarly, do you have an idea about how it should interact with "garbage
> collection" story?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/fsprojects/Paket/pull/1619/files/3eda83abb200591365d4aac47cbe30d908b111ba#r62631374>
>